Utils/postUtils.js

```diff
@@ -0,0 +1,16 @@
+import axios from "axios";
+
+
+const NCnewsAPI = axios.create({
+  baseURL: "https://nc-news-tm65.onrender.com",
+});
+
+export const PostCommentByArticle = (comment, id) => {
+  return NCnewsAPI.post(`/api/articles/${id}/comments`, comment)
+    .then((res) => {
+      return res.data;
+    })
+    .catch((err) => {
```

Comment:
There should be no catch blocks in your utils/api calls file. Your back end _should_ be handling that.